app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -18,17 +18,17 @@ def create
       req_body_obj = request_body_hash.is_a?(String) ? JSON.parse(request_body_hash) : request_body_hash
       form4142 = req_body_obj.delete('form4142')
       sc_evidence = req_body_obj.delete('additionalDocuments')
+      zip_from_frontend = req_body_obj.dig('data', 'attributes', 'veteran', 'address', 'zipCode5')
```

Comment:
Would there ever be a reasonable situation where there would be a `nil` zip code in the request body object?